src/components/auth/AuthForm.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button } from 'primereact/button';
+import { InputText } from 'primereact/inputtext';
+import { Password } from 'primereact/password';
+import { useState } from 'react';
+import { loginUser } from './service';
+import '../../styles/components/Auth.styles.scss';
+
+function AuthComponent() {
```

Comment:
Название компонента должно полностью совпадать с название файла. Файлы css называем идентично названию компонента только с модулем: `AuthForm.module.scss`
```suggestion
function AuthForm() {
```

Reply:
Переделаю

Просто ты до этого писал *.styles.scss
![image](https://github.com/user-attachments/assets/bd42375c-8aa3-435c-93b7-2d78948361b8)

Reply:
Переделаю

Просто ты до этого писал *.styles.scss
image

Reply:
Привет! Я это и имел ввиду, видимо как-то не очень доступно объяснил)